src/queries/useTrendingMovies.ts

```diff
@@ -0,0 +1,15 @@
+import { keepPreviousData, useQuery } from '@tanstack/react-query';
+import { trendingMovies } from '../services/movie-service';
+import { type MovieResponse } from '../types/movies/MovieResponse';
+import { delay } from '../utils/async/delay';
+
+export function useTrendingMovies(page = 1) {
```

Comment:
`useTrendingMovies` never gets page as an argument in the app, so it always equal 1.